fdbserver/worker.actor.cpp

```diff
@@ -377,6 +377,11 @@ struct DiskStore {
 std::vector< DiskStore > getDiskStores( std::string folder, std::string suffix, KeyValueStoreType type) {
```

Comment:
This function is called once per store type, and all store types except for one do not accept directories as data locations so there should be a parameter here which enables the search for directories.  Also, since the store type which does use directories also does not use files, perhaps there should be two bool arguments, `findFiles` and `findFolders`, and the calls to `getDiskStores()` can specify which apply.

Reply:
I used an enum instead of two bools, but the code now only checks for files for not RocksDB and only checks for directories with FDB.